src/components/dashboard/Dashboard.jsx

```diff
@@ -141,8 +142,8 @@ class Dashboard extends React.Component {
   _renderStatisticsTile() {
     return this._isAdmin() ? (
       <Col md={3} className="dashboard-sector">
-        <DashboardTile onClick={this.props.handlers.showStatistics}>
```

Comment:
The functionality should be the same if ${ANALYTICS_URL} is same !

Reply:
![image](https://github.com/kbss-cvut/record-manager-ui/assets/3962237/b902a5d5-71f7-4c41-94cb-cb1da409910c)